### ISSUE TITLE
Horizontal scrolling in hourglass-chart broken/wrong focus?

### ISSUE THREAD

**Issue by someone-somenet-org** [fisharebest/webtrees#5032]
Arrow keys and dragging doesnt work horizontally, but vertically its fine for some reason.

You have to click into the pane or the scrollbar to be able to scroll with the arrow keys, it doesnt work with dragging.

Please consider to make the hourglass-chart full width and make the scrollbar react to dragging and arrow keys.

**Comment by arbor95** [comment]
did that ever work?

**Comment by someone-somenet-org** [comment]
> did that ever work?

Yes. Years ago the chart was not "painted" onto a sub-div, but directly onto the page-body and you scrolling horizontally would lead to the menu at the top to scroll away.

**Comment by fisharebest** [comment]
> Arrow keys and dragging doesnt work horizontally, but vertically its fine for some reason.

Dragging works fine for me on touch-screens.

With a mouse, shift-scroll-wheel scrolls horizontally.

> You have to click into the pane or the scrollbar to be able to scroll with the arrow keys

This is expected.  If the focus is on the main page, then it has no horizontal scroll.  Only the inner div has horizontal scroll.

You need to tab to the required area.

If there was more than one area with horizontal scroll, then you would need a way to choose the area to scroll.

This is working as intended (so not a bug), and I think it follows conventional behaviour for web pages.

You said that dragging does not work for you.  What browser/device are you using?
